packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart

```diff
@@ -434,6 +435,15 @@ class _ResidentWebRunner extends ResidentWebRunner {
             return 1;
           }
           device.generator.accept();
+          globals.logger.printTrace('caching compiled dill...');
```

Comment:
Instead of duplicated here and in the hot runner, can this be inherited by both from the base class?

Reply:
Done